src/routes/main-route.jsx

```diff
@@ -1,8 +1,33 @@
+import Image from "../components/Image/Image";
```

Comment:
After you make the change to add `index.js` files to each of the subdirectories, this should be changed to the following:
```
import { Image, Header, MovieCardList, PageTitle } from '../components';
```